scapy/tools/automotive/isotpscanner.py

```diff
@@ -0,0 +1,197 @@
+#! /usr/bin/env python
+
+# This file is part of Scapy
+# See http://www.secdev.org/projects/scapy for more information
+# Copyright (C) Nils Weiss <nils@we155.de>
+# Copyright (C) Alexander Schroeder <alexander1.schroeder@st.othr.de>
+# This program is published under a GPLv2 license
+
+from __future__ import print_function
+
+import getopt
```

Comment:
FYI, [argparse](https://docs.python.org/2/howto/argparse.html) will have save you some time & code.

Reply:
That would make me feel pretty dumb :smile: https://github.com/secdev/scapy/pull/2058#discussion_r296674975
In the previous state of the PR, it added an extra dependency